lib/spack/docs/tutorial/examples/test.yaml

```diff
@@ -0,0 +1,2231 @@
+---
```

Comment:
The file is too long -- if it's this long it should really just be generated.  Can you just remove this and link to the docs in the command help where this is referenced?

Also, `lib/spack/docs/tutorial/examples`  is really *just* for our [SC'16 tutorial](https://spack.readthedocs.io/en/latest/tutorial_sc16.html).  Examples like this should probably get their own directory separate from that.